### ISSUE TITLE
"Osprey" Helicopter crash loot is too good.

### ISSUE THREAD

**Issue by AdonaiJr** [CleverRaven/Cataclysm-DDA#25207]
I recently found the newly added Osprey helicopter crash. While I think it was an amazingly interesting find (taking me to wikipedia to read more about this somewhat different aircraft), I felt that the loot was too good and too easy to get.

It gave me almost 4 full sets of military clothes, MBR vest, M4A1 with additional magazine, Beretta (and more ammo), combat knife, holster, boots, etc. 4 of each one of these things among different soldier corpses. They were also intact.

I suggest a better balance, like:

- Reducing the spawn chance;
- Giving chance to spawn damaged items;
- Chance to spawn "soldier type" zombies (so you must _earn_ your loot);

That said, I appreciate a lot the work put on to give life to these new helicopter crashes!

**Comment by CoroNaut** [comment]
I always find at least one zombie at the crash sites I find. Perhaps your zombie wandered off? They are the zombie military pilots.

**Comment by Inglonias** [comment]
Starting the game with heli ctashes has taught me that there can be as many as half a dozen soldiers and two bio operators at the crash site.

**Comment by Regularitee** [comment]
The too-much-loot issue is probably due to the fact all aircraft currently spawn with a full crew (unlike every other vehicle in game), which in turn almost guarantees lots of military loot per crash site. See also #25143

**Comment by FulcrumA** [comment]
True, some of those things were kinda haphazard initial version or workarounds. I plan to test and improve a few things, updating loot at least a bit once real life will let me spend more time on it (though lesser changes I will try to make soon). Some of it should be however already in - soldier zombies, including more dangerous military ones like bio-operators @Inglonias mentions are a thing.

**Comment by John-Candlebury** [comment]
Honestly the amounts are what I would  expect of a military crash IRL.

The loot should however be all heavily damaged.

**Comment by FulcrumA** [comment]
There will be a bit less of it in a way that the chance to find gutted up body with its clothes will be lower, but you will be able to find greater variety of things, nearly like how it is with regular soldier zombies. The stuff will be damaged though. I've got the json part of it ready but certain.. sillyness in the code waits for #25233 to get through before I can continue with the planned improvement.
